packages/core/src/Activatable.ts

```diff
@@ -0,0 +1,10 @@
+import { isActivatable } from "./util/is-activatable.js";
+
+/**
+ * Interface for activatable objects.
+ * 
+ * @see {@link isActivatable}
```

Comment:
I don't think this (or the import) are really needed here. So we could simplify that.